### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by ruggero-nardi**
Done up to exercise 1-iv

### CODE REVIEW

File: Exercise-1/Expenses.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Expenses</title>
+    </head>
+    <body>
+        <table>
+            <tr>
+                <td>DESCRIPTION</td>
```
Comment (troyax) [inline Exercise-1/Expenses.html]: Are you sure `td` is the right tag for table headers?

File: Exercise-1/Expenses.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Expenses</title>
+    </head>
+    <body>
+        <table>
+            <tr>
```
Comment (troyax) [inline Exercise-1/Expenses.html]: Try using tags to split content from headers. That is useful when writing semantic html.

File: Exercise-1/Audio-img-video.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Image, Audio and Video</title>
+    </head>
+    <body>
+        <div id="image">               
+            <figure>
+                <img src="lion.jpg" alt="Young lion on a rock" width="642">
```
Comment (troyax) [inline Exercise-1/Audio-img-video.html]: Is not a good practice to use attributes to style stuff. Please use CSS for that

File: Exercise-1/Audio-img-video.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            </figure>
+        </div>
+        <div id="audio">           
+            <audio controls>
+                <source src="music.mp3" type="audio/mpeg">  
+                Audio not supported by this browser
+            </audio>
+        </div>
+        <div id="video">
+            <video width="360" height="240" controls>
```
Comment (troyax) [inline Exercise-1/Audio-img-video.html]: Same here

File: Exercise-1/E1-TheBasicsOfHTML.html [hunk 5]
```diff
@@ -1,17 +1,22 @@
...
-            <li>Go to the supermarket</li>
-            <li>Walk the dog</li>
-            <li>Study</li>
-            <li>Go to class</li>
-            <li>Clean the house</li>                        
-        </ul>
+        <section>
+            <article>
+                <h1>My todo list</h1>
+                    <ul>
```
Comment (troyax) [inline Exercise-1/E1-TheBasicsOfHTML.html]: The identation of the `ul` as also the `li` might be different to be consistent. But we will talk about that today.

**Review by troyax — COMMENTED** [review]
Some other comments but almost ready :)

File: Exercise-1/E1-TheBasicsOfHTML.html (hunk 5 above)
Comment (ruggero-nardi) [reply Exercise-1/E1-TheBasicsOfHTML.html]: Oh i see, i added an extra indentation space after the h1. I'll fix that.

**Review by troyax — APPROVED** [review]

**Pull request merged**